Formula/git-bug.rb

```diff
@@ -0,0 +1,17 @@
+class GitBug < Formula
+  desc "Distributed, offline-first bug tracker embedded in git, with bridges"
+  homepage "https://github.com/MichaelMure/git-bug"
+  url "https://github.com/MichaelMure/git-bug/archive/0.7.1.tar.gz"
+  sha256 "78a6c7dee159cdad4ad69066d6d8fc4b7c259d5ea6351aaf6709b6ac03ff3d2f"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "build"
+    bin.install "git-bug"
+  end
+
+  test do
+    assert_includes shell_output("#{bin}/git-bug --version"), "git-bug version"
```

Comment:
We're going to need a test that exercises some functionality of the app, rather than just a `--version` check. From a quick read through the README, maybe you could try [one of the usage examples](https://github.com/MichaelMure/git-bug#cli-usage) and match on some error messages?

Reply:
Done, I could add only tests on the basic commands (others open interactive UIs) that all fail with the same error because they must run from a git repo. Still useful to check if the binary works.

Reply:
Fixed formatting, CI triggered again.